pymc3/step_methods/hmc/quadpotential.py

```diff
@@ -94,7 +94,7 @@ def update(self, sample, grad, tune):
         """
         pass
 
-    def raise_ok(self):
+    def raise_ok(self, vmap):
```

Comment:
Maybe safer to make it `vmap=None`?

Also, I feel like we should explain why there's an odd argument in a docstring.